packages/pf-codemods/index.js

```diff
@@ -81,6 +81,9 @@ function getRulesToRemove(options) {
 async function runCodemods(path, otherPaths, options) {
   const prefix = "@patternfly/pf-codemods/";
 
+  // data-codemods-cleanup rule won't be included in the recommended rule set
+  delete configs.recommended.rules[prefix + "data-codemods-cleanup"];
+
```

Comment:
Could you not just add the rule to the betaRules array in the customization file instead of adding logic to this file?

Reply:
Oh, yeah, I did not read the betaRuleNames description, the name "beta" confused me a bit

Reply:
FWIW Austin and I had a convo about the naming of the array, though I don't think we settled on whether to change it (or to what if so). Maybe post v6 we can have a brainstorm session about that + anything else we'd like to see changed in the repo